### ISSUE TITLE
Option for line width and/or font size control when using 'size = ___' for geom_textsmooth()

### ISSUE THREAD

**Issue by adamkemberling** [AllanCameron/geomtextpath#65]
Hi all, thank you for the excellent package. Everything is documented well and its easy to drop these geoms into existing work. I did have a question or a feature that might be useful, and that is the option to set the line width and the text size when using `geom_textsmooth()` Currently `size = ____` can be used within the function to adjust font size, which blocks the ability to change line thickness as one would with `ggplot2::geom_smooth()`. I can think of cases where one might want to have the ability to adjust both independently and just wanted to raise the idea. Thanks again for sharing this excellent package.

**Comment by teunbrand** [comment]
Hi there,

The size of the line is controlled by the `linewidth` aesthetic. We had to rename this because in case of a text + path, it was ambiguous to which `size` should apply. We opted that it should apply to the text in the end, so the line has a different aesthetic, which we tried to explain in the [aesthetics vignette](https://allancameron.github.io/geomtextpath/articles/aesthetics.html#aesthetics).
